### ISSUE TITLE
Review use of `CA1837` warning - Use 'Environment.ProcessId'

### ISSUE THREAD

**Issue by RussKie** [dotnet/winforms#3982]
To update the underlying SDK/toolsets to 5.0-RC1 we had to suppress `CA1837 // Use 'Environment.ProcessId'` in https://github.com/dotnet/winforms/pull/3959/commits/9dfaaa152b889f2040f646697dfb31c77d7058e8.

https://github.com/dotnet/winforms/blob/ec688d7d10409fe3566e8eebaa066b962f38d9cb/src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewSubItem.ListViewSubItemAccessibleObject.cs#L115-L117

The suppression needs to be reviewed, and corrected.

**Comment by swapniljha001** [comment]
Hello @RussKie what is the issue here, how may I resolve this issue?

**Comment by RussKie** [comment]
The code needs to be corrected so we can remove the warning suppression. It should be fairly straight forward, but the change must be validate using accessibility tools, such as Narrator. 
To boost the confidence level you could add 2 animations - before and after the change.

@vladimir-krestov @SergeySmirnov-Akvelon can you please describe validation steps?

**Comment by vladimir-krestov** [comment]
To check the correctness of using `Environment.ProcessId` I made this:

![image](https://user-images.githubusercontent.com/49272759/94255333-bea1a880-ff30-11ea-8eeb-e786f9a1ce78.png)

Then I run Inspect and check these properties:

![image](https://user-images.githubusercontent.com/49272759/94255797-7c2c9b80-ff31-11ea-867c-f4a11cce24f5.png)

So we can use `Environment.ProcessId` instead `Process.GetCurrentProcess().Id` without problem.

**Comment by hughbe** [comment]
Should we consolidate `Kernel32. GetCurrentProcessId`, `Environment.ProcessId` and `Process.GetCurrentProcess().Id`

**Comment by RussKie** [comment]
> Should we consolidate `Kernel32. GetCurrentProcessId`, `Environment.ProcessId` and `Process.GetCurrentProcess().Id`?

I'd go for `Environment.ProcessId`. 
What are your preference @JeremyKuhne  @hughbe @weltkante?

**Comment by JeremyKuhne** [comment]
`Environment.ProcessId` is a simple cached value so it is probably the best given it is the ".NET" way of doing it.

**Comment by AraHaan** [comment]
Since this is .NET 6 is it alright if since I am working on #3983 as well to bundle this with the changes that this would need in the same pr or different pr?

Note: I will use github to make the commits for now since I am currently on my mac.

Also since ``Environment.ProcessId`` is the same as ``Kernel32.GetCurrentProcessId`` should the p/invoke to that function get nuked entirely for ``Environment.ProcessId``, and also change ``Process.GetCurrentProcess().Id`` to that as well (could possibly remove the usage of ``System.Diagnostics.Process`` a lot indirectly however.

An issue with doing it on browser on github is that then I would not be able to catch when unneeded usings are in the modified code.

**Comment by AraHaan** [comment]
https://github.com/dotnet/winforms/search?q=Kernel32.GetCurrentProcessId

https://github.com/dotnet/winforms/search?q=Kernel32.GetCurrentProcessId

**Comment by AraHaan** [comment]
I think the ``Process.GetCurrentProcess.Id`` could be at least replaced with ``Environment.ProcessId`` but turned into a string.

**Comment by weltkante** [comment]
I wouldn't rely on github search, its ok as a first start but in my experience it often returns less matches than a normal plaintext search, especially if you search for more (or less) than a single word.

**Comment by AraHaan** [comment]
Yep, github search needs to be enhanced to be more accurate on finding results in code.
